planner/core/rule_aggregation_elimination.go

```diff
@@ -54,22 +55,29 @@ func (a *aggregationEliminateChecker) tryToEliminateAggregation(agg *LogicalAggr
 	}
 	schemaByGroupby := expression.NewSchema(agg.GetGroupByCols()...)
 	coveredByUniqueKey := false
+	var uniqueKey expression.KeyInfo
 	for _, key := range agg.children[0].Schema().Keys {
 		if schemaByGroupby.ColumnsIndices(key) != nil {
 			coveredByUniqueKey = true
+			uniqueKey = key
 			break
 		}
 	}
 	if coveredByUniqueKey {
 		// GroupByCols has unique key, so this aggregation can be removed.
 		if ok, proj := ConvertAggToProj(agg, agg.schema); ok {
 			proj.SetChildren(agg.children[0])
+			opt.appendStepToCurrent(agg.ID(), agg.TP(),
+				fmt.Sprintf("%s is unique key", uniqueKey.String()),
```

Comment:
Will this expose the user's data information in some ways?

Reply:
No, there is only column description, not real data.